src/app/components/Drbb/DrbbItem.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import appConfig from '../../data/appConfig';
+import {
+  authorQuery,
+  formatUrl,
+} from '../../utils/researchNowUtils';
+
+const DrbbItem = (props) => {
+  const { work } = props;
+  const {
+    agents,
+    title,
+    editions,
+  } = work;
+
+  const {
+    environment,
+  } = appConfig;
+
+  const drbbFrontEnd = appConfig.drbbFrontEnd[environment];
+
+  const edition = editions[0];
+  const item = edition.items[0];
+
+  const authorship = () => {
+    const authors = agents.map((agent, i) => [
+      (i > 0 ? ', ' : null),
+      <Link
+        to={{
+          pathname: `${drbbFrontEnd}/search`,
+          query: authorQuery(agent),
```

Comment:
I think this query will also fail as it will be structured for POST but made with GET

Reply:
This one works. It's not in their docs, but with 'react-router''s `Link` `to.query` accepts an object and does the work of stringify and encoding. 

What does `showQueries` do by the way?